Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Generic Ranged Weapons

#### Changelog:
- Added generic ranged artifacts


#### How to test:

- Check that artifact stats are correct. Use [this](https://docs.google.com/spreadsheets/d/1fPiP2HnYhC80hOelgUdeuvItH__uX5H27NFSOwH-tyA/edit#gid=1787561420&range=48:50) and [this](https://docs.google.com/spreadsheets/d/1xDjRHfG-ovKVcEQVfv02BsK3j0dgwS2rfH7KJ4GirA0/edit#gid=1700305504&range=5:54) spreadsheets.
- Proofread [this](https://docs.google.com/spreadsheets/d/1FVVfKRCmTzPfiqoKvcfllqOhR613hpCzm2XJ8Gk1moU/edit#gid=15614703&range=1:91)
- Check that names are assigned correctly. Don't forget that there is 2 names variation - `[№1] [Type] of [№3] [№4]` (e.g. `Marksman's Bow of Wisp Vanquishing`) and `[Type] of [№2] [№3] [№4]` (e.g. `Rifle of Scary Octopus Repelling`)
